cli/__snapshots__/cli_spec.ts.js

```diff
@@ -75,7 +75,7 @@ exports['shows help for run --foo 1'] = `
     -C, --config-file <config-file>                             path to script file where configuration values are set. defaults to "cypress.config.{js,ts,mjs,cjs}".
     --e2e                                                       runs end to end tests
     -e, --env <env>                                             sets environment variables. separate multiple values with a comma. overrides any value in cypress.config.{js,ts,mjs,cjs} or cypress.env.json
-    --group <name>                                              a named group for recorded runs in Cypress Cloud
+    --group <n>                                                 a named group for recorded runs in Cypress Cloud
```

Comment:
I don't see the reason for this change - this seems less descriptive to me.

Reply:
I originally thought that `<n>` was the original but I think the AI on the conversion just decided to rewrite it? So bizarre because this is the original https://github.com/cypress-io/cypress/blob/develop/cli/lib/cli.js#L244C21-L244C25. Ill change it back

Reply:
@jennifer-shehane fixed in 5d027996af